functions/Find-DbaDisabledIndex.ps1

```diff
@@ -0,0 +1,162 @@
+
+        .PARAMETER Confirm
+            If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Tags: Indexes
+            Author: Jason Squires, sqlnotnull.com
+
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            https://dbatools.io/Find-DbadisabledIndex
+
+        .EXAMPLE
+            Find-DbadisabledIndex -SqlInstance sql2005
+
+            Generates the SQL statements to drop the selected disabled indexes on server "sql2005".
+
+        .EXAMPLE
+            Find-DbadisabledIndex -SqlInstance sqlserver2016 -SqlCredential $cred
+
+            Generates the SQL statements to drop the selected disabled indexes on server "sqlserver2016", using SQL Authentication to connect to the database.
+
+        .EXAMPLE
+            Find-DbadisabledIndex -SqlInstance sqlserver2016 -Database db1, db2
+
+            Generates the SQL Statement to to drop selected indexes in databases db1 & db2 on server "sqlserver2016".
+
+        .EXAMPLE
+            Find-DbadisabledIndex -SqlInstance sqlserver2016
+
+            Generates the SQL statements to drop selected indexes on all user databases.
+
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    Param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]
+        $SqlCredential,
+        [Alias("Databases")]
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [switch]$NoClobber,
+        [switch]$Append,
+        [switch][Alias('Silent')]$EnableException
+    )
+
+    begin {
+        $sql = "
+        SELECT DB_NAME() AS 'DatabaseName'
+        ,s.name AS 'SchemaName'
+        ,t.name AS 'TableName'
+        ,i.object_id AS ObjectId
+        ,i.name AS 'IndexName'
+        ,i.index_id as 'IndexId'
+        ,i.type_desc as 'TypeDesc'
+        FROM sys.tables t
+        JOIN sys.schemas s
+            ON t.schema_id = s.schema_id
+        JOIN sys.indexes i
+            ON i.object_id = t.object_id
+        WHERE i.is_disabled = 1"
+    }
+    process {
+        Write-Message -Level Output -Message "Attempting to connect to Sql Server."
+        $server = Connect-SqlInstance -SqlInstance $SqlInstance -SqlCredential $SqlCredential -MinimumVersion 9
```

Comment:
Still need to address a `foreach` loop to support multiple instances being passed in, or piped.

Also wrap the connect attempt in try/catch. This is the pattern we use for the other commands:

```powershell
            try {
                Write-Message -Level Verbose -Message "Connecting to $instance."
                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
            }
            catch {
                Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
            }
```